### ISSUE TITLE
gcc@9.3.0-1: Download failed

### ISSUE THREAD

**Issue by RobViren** [ScoopInstaller/Main#1471]
Install of gcc on main bucket not working.
```powershell
scoop install gcc
Installing 'gcc' (9.3.0-1) [64bit]
Unable to connect to the remote server
URL http://repo.msys2.org/mingw/x86_64/mingw-w64-x86_64-binutils-2.34-1-any.pkg.tar.xz is not valid

scoop --version
Current Scoop version:
6df80c1a (HEAD -> master, origin/master, origin/HEAD) fix(tests): Force pester v4 (#4040)
'main' bucket:
1cc739df (HEAD -> master, origin/master, origin/HEAD) imagemagick: Update to version 7.0.10-33
```

**Comment by jakubgs** [comment]
A mirror was set up in https://github.com/msys2/MINGW-packages/issues/7084, but that's probably temporary.

**Comment by habamax** [comment]
![image](https://user-images.githubusercontent.com/234774/95473240-92544600-098c-11eb-8502-27e8591854d3.png)

**Comment by jakubgs** [comment]
See: https://github.com/msys2/MINGW-packages/issues/7084#issuecomment-705456828

Maybe using a different server as done in https://github.com/ScoopInstaller/Main/pull/1476 is a good idea.

**Comment by wstein** [comment]
http://repo.msys2.org/mingw/x86_64/ is up again, so there is no need to fix the url.

But the versions still must be fixed, as msys dropped some version and of cause also the mirror. ;-)

**Comment by cortexcompiler** [comment]
I see a similar issue with the `openssh` package, should that be another issue?
```
Installing 'openssh' (8.2p1-1) [64bit]
bash-4.4.023-2-x86_64.pkg.tar.xz (1.9 MB) [==================================================================] 100%
Checking hash of bash-4.4.023-2-x86_64.pkg.tar.xz ... ok.
gcc-libs-9.3.0-1-x86_64.pkg.tar.xz (1.2 MB) [================================================================] 100%
Checking hash of gcc-libs-9.3.0-1-x86_64.pkg.tar.xz ... ok.
The remote server returned an error: (404) Not Found.
URL http://repo.msys2.org/msys/x86_64/heimdal-7.7.0-1-x86_64.pkg.tar.xz is not valid
```
Scoop Version info:
```
scoop --version
Current Scoop version:
6df80c1a (HEAD -> master, origin/master, origin/HEAD) fix(tests): Force pester v4 (#4040)

'extras' bucket:
f803f7c66 (HEAD -> master, origin/master, origin/HEAD) firefox-nightly: Update to version 84.0a1.20201021095128

'java' bucket:
ecaa54bf (HEAD -> master, origin/master, origin/HEAD) corretto11: Update to version 11.0.9.11.1

'main' bucket:
8d4cc854 (HEAD -> master, origin/master, origin/HEAD) vim-nightly: Update to version 8.2.1873
```

**Comment by wstein** [comment]
> I see a similar issue with the `openssh` package, should that be another issue?

Thanks for reporting, but there is already a bug report: #1492
